packages/office-ui-fabric-react/src/components/Overlay/Overlay.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { IOverlayStyleProps, IOverlayStyles } from './Overlay.types';
+import {
+  IStyle,
+  ITheme,
+  HighContrastSelector,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IOverlayStyleProps
+): IOverlayStyles => {
+  const {
+    className,
+    theme,
+    isNone,
+    isDarkThemed,
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  return ({
+    root: [
+      'ms-Overlay',
+      {
+        backgroundColor: palette.whiteTranslucent40,
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+
+        selectors: {
+          [HighContrastSelector]: {
+            border: '1px solid WindowText',
+          }
+        }
+      },
+
+      isNone && {
```

Comment:
Process question: From what I can tell this was never actually applied before (just an unused SCSS class), and you went ahead and applied that as an optional prop.  Do we want to do this, or assume abandoned functionality was abandoned for a reason.  Like in this case does it serve a purpose or is it just a superfluous prop taking up bit space.

Reply:
Good question. @battletoilet